integr8ly/integreatly-operator
# retrieve SMTP from envar

# Description
In case alertmanager-application-monitoring secret is not present, `SMTP from` address will be retrieved from environmental variable. This will prevent installation to go to the failed stage in case the secret is missing. 

## Verification steps
1. Run a fresh installation on a cluster 
2. Once operator goes in the `products` stage enter the following command: 
```
watch "oc delete secret alertmanager-application-monitoring -n redhat-rhoam-middleware-monitoring-operator"
```
3. Verify installation completes 

## Type of change

## Checklist
